pygradle-plugin/src/test/groovy/com/linkedin/gradle/python/extension/DefaultPythonDetailsUnixTest.groovy

```diff
@@ -82,15 +82,6 @@ class DefaultPythonDetailsUnixTest extends Specification {
         details.getSystemPythonInterpreter().path.endsWith("3.5")
     }
 
-    @Requires({ new File('/usr/bin/python2.7').exists() && new File('/usr/bin/python2.6').exists() })
-    def "interpreterPath with systemPython set"() {
-        when: "we have old systemPython setting"
-        details.pythonVersion = '2.6'
```

Comment:
python n00b question - is there a different scenario where the system vs. interpreter versions could vary outside of 2.6 & 2.7?

Reply:
We are going to lose a test here.
Basically, the idea is to confirm that we can change a default interpreter.
So, we were doing that with change from the default 2.6 to 2.7.
Perhaps we should now test that we can change it from the default 2.7 to one of the Python 3 versions, for example 3.5 or 3.6.
That way we keep the test and its intent, but adapt it to our change here.
We'd need to adapt `@Requires` accordingly.

Reply:
@zvezdan Good point, thanks.  I'll fix it exactly as you say.